addons/mail/static/src/models/composer/composer.js

```diff
@@ -221,10 +221,6 @@ function factory(dependencies) {
          * @returns {mail.partner[]}
          */
         _computeRecipients() {
-            if (this.thread && this.thread.model === 'mail.channel') {
-                // prevent from notifying/adding to followers non-members
-                return [['unlink-all']];
-            }
```

Comment:
Hello, just to let you know the fix to this has been merged, so you should rebase to restore proper mentions in channels.

Reply:
Also don't forget to add a test for the new highlight feature.